alerts_consumer.go

```diff
@@ -201,18 +192,35 @@ func (c *AlertsConsumer) SendBatch(batch [][]byte, tag string) error {
 		}
 	}
 
-	if err := c.sfxSink.AddDatapoints(context.TODO(), pts); err != nil {
-		if err.Error() == "invalid status code 400" { // internal buffer full on sfx's side
-			return kbc.PartialSendBatchError{ErrMessage: "failed to add datapoints: " + err.Error(), FailedMessages: batch}
+	var err error
+	sleep := 50
```

Comment:
I think this is clearer if we make it a `time.Duration` and do the multiplication with `time.Millisecond` here